test/ServiceBus.Tests/PlugableQueueBalancerTests/LeaseManagerGrain.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Orleans;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using Orleans.Streams;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tester.StreamingTests
+{
+    //one lease manager grain per stream provider, so its key is stream provider name
+    public interface ILeaseManagerGrain : IGrainWithStringKey
```

Comment:
Just used in tests.
The test is a small probe work to prototype LeaseManager based queue balancer, mainly to make sure it can access most dependency object it needs to access.